components/productions/index/events.vue

```diff
@@ -64,11 +64,12 @@
         >{{ errorMessage }}
       </pub-alert>
       <pub-panel>
-        <pub-list>
+        <pub-list aria-label="events">
```

Comment:
Will this be read out loud by screen readers? Should we make it translatable? Probably not that big a deal for an internal feature like productions, but want to make sure we don't discover later that we have a lot of `aria-label` attributes throughout the app that need to become translatable.

Reply:
(Also for the other aria-label attributes added below that are not based on user input)